### PR TITLE
Add support for displaying the server communication status as well

### PR DESCRIPTION
**Pull request by shankari**
If we are not getting data from users, the reasons can be:
- problems with the sensing, or
- problems with the communication to the server

The current information in the dashboard shows how much data we have received,
but does not help with troubleshooting missing data.

With these new columns, we have additional information on when the users last
communicated with the server.

If the users have not communicated for a long time, the problem is likely to be with server communication.
If the users have been communicating regularly, but still don't have any data,
the problem is likely to be with sensing.

### CODE REVIEW

**Comment by shankari** [comment]
This is what it looks like for me

Screen Shot 2020-11-30 at 4 18 38 PM

**Comment by asiripanich** [comment]
any reason not to display time as well?

**Comment by shankari** [comment]
No. The time didn't display for me by default although I used `lubridate::as_datetime`. And the time wasn't super important, so I didn't debug further to fix it.

File: R/utils_tidy_data.R [hunk 1]
```diff
@@ -161,6 +161,29 @@ summarise_trips = function(participants, trips) {
...
+#' @param participants the output from `tidy_participants()`.
+#' @param trips the output from `tidy_cleaned_trips()`.
+#'
+#' @return a data.table.
+#' @export
+summarise_server_calls = function(participants, server_calls) {
+  summ_calls <-
+    server_calls %>%
+    data.table::setDT(.) %>%
+    .[, date := lubridate::date(lubridate::as_datetime(ts))] # adds the date of local datetime of trip
```
Comment (asiripanich) [inline R/utils_tidy_data.R]: probably this `lubridate::date(` causes the time component to be striped.

**Pull request merged**